test/test_rw_fortran.F90

```diff
@@ -73,34 +74,34 @@ END FUNCTION real_eq
   DO i = 1, COMMAND_ARGUMENT_COUNT()
      CALL GET_COMMAND_ARGUMENT(i,arg,len,status)
      IF (status .NE. 0) THEN
-        WRITE (*,*) 'get_command_argument failed: status = ', status, ' arg = ', i
+        WRITE (ERROR_UNIT,*) 'get_command_argument failed: status = ', status, ' arg = ', i
```

Comment:
@brtnfld are you ok with changes to output error messages from `WRITE (*,*)` here to `WRITE (ERROR_UNIT, *)`? In particular, take note that messages for when things pass (lines 291/294) are written to `OUTPUT_UNIT`.

Reply:
I'm ok with that.